lib/message.js

```diff
@@ -16,9 +16,9 @@ var $ = bitcore.util.preconditions;
  * constructs a new message to sign and verify.
  *
  * @param {String} message
- * @returns {Message}
```

Comment:
Correct it can be inferred since it's a constructor (though often `new` isn't used, so it may be helpful to have this anyways).